docs/source/delta-batch.md

```diff
@@ -1196,6 +1196,13 @@ You can also set `delta.`-prefixed properties during the first commit to a Delta
 
 See also the [_](/table-properties.md).
 
+## Syncing table schema and properties to the Hive metastore
+
+You can enable asynchronous syncing of table schema and properties to the metastore by setting `spark.databricks.delta.catalog.update.enabled` to `true`. Whenever the Delta client detects that either of these two were changed due to an update, it will sync the changes to the metastore. The config `spark.databricks.delta.catalog.update.threadPoolSize` can be optionally used to control the size of the threadpool that is used for these asynchronous updates.
```

Comment:
```suggestion
You can enable asynchronous syncing of table schema and properties to the metastore by setting `spark.databricks.delta.catalog.update.enabled` to `true`. Whenever the Delta client detects that either of these two were changed due to an update, it will sync the changes to the metastore.```

`spark.databricks.delta.catalog.update.threadPoolSize` is an internal conf - lets avoid talking about it in docs. Also for all practical purposes, we don't expect users to play with it.
This is an